packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -447,6 +450,25 @@ export class FileStorage implements DStore {
         if (n.stub) {
           return;
         }
+        if (n.body.length >= CONSTANTS.DENDRON_FILE_TOO_BIG) {
+          this.logger.info({
+            ctx,
+            msg: "Note too large, skipping",
+            note: NoteUtils.toLogObj(n),
+            length: n.body.length,
+          });
+          errors.push(
+            new DendronError({
+              message: `Note "${n.fname}" in vault "${VaultUtils.getName(
+                n.vault
+              )}" is too large, some features like backlinks may not work correctly for it.`,
+              severity: ERROR_SEVERITY.MINOR,
+            })
+          );
```

Comment:
nitpick: Should this be an error? We are explicitly limiting part of the feature and otherwise the plugin is working as intended so I think a warning might be sufficient. With an error the user may think they did something wrong and it needs to be fixed.

Also, IIRC a lot of the people who encounter issues with big files were curious what _too big_ meant. It may be a nice touch to add a button in the toast notification that links to a doc that tells you what the limit is.

Reply:
The error is to pass the warning from the engine to the editor and have it be displayed. I didn't want to make a bigger change to pass warnings seperately, but we can do it if there's a benefit.

Agree with communicating how big is too big. Maybe I can add that to the warning message?

Reply:
We do have the severity argument so maybe we can do something where the toast notification is actually made depending on the severity that gets passed, but this could change the behavior of the notification in other places so it probably should be more thought out before we do this 😅  Did we have an SOP that goes over error message conventions or something?

> Agree with communicating how big is too big. Maybe I can add that to the warning message?
Sounds good